src/MongoDB/Driver/Manager.cpp

```diff
@@ -522,6 +535,158 @@ static bool hippo_mongo_driver_manager_apply_ssl_opts(mongoc_client_t *client, c
 	return 1;
 }
 
+void command_started(const mongoc_apm_command_started_t *event)
+{
+	Class *cls = Unit::lookupClass(s_MongoDBDriverManager__Dispatcher.get());
+	Func  *m   = cls->lookupMethod(s_MongoDBDriverManager_dispatchCommandStarted.get());
+	static Class *c_event;
+	Variant    result;
+	ObjectData *obj_context = (ObjectData*) mongoc_apm_command_started_get_context(event);
+	MongoDBDriverManagerData* data = Native::data<MongoDBDriverManagerData>(obj_context);
+
+	auto prop = cls->lookupSProp(s_MongoDBDriverManager_subscribers.get());
+	auto val  = cls->getSPropData(prop);
+
+	if (!val) {
+		return;
+	}
+
+	Array subscribers = tvAsVariant(val).toArray();
+
+	if (subscribers.size() < 1) {
+		return;
+	}
+
+	c_event = Unit::lookupClass(s_MongoDriverMonitoringCommandStartedEvent_className.get());
+	Object o_event = Object{c_event};
+	o_event->o_set(s_MongoDriverMonitoringCommandEvent_commandName, Variant(mongoc_apm_command_started_get_command_name(event)), s_MongoDriverMonitoringCommandEvent_className);
+	o_event->o_set(s_MongoDriverMonitoringCommandEvent_server, Variant(hippo_mongo_driver_server_create_from_id(data->m_client, mongoc_apm_command_started_get_server_id(event))), s_MongoDriverMonitoringCommandEvent_className);
```

Comment:
Unlike https://github.com/mongodb/mongo-php-driver/pull/466, you don't store the `mongoc_client_t` on the event object because you're constructing the `MongoDB\Driver\Server` object immediately. Is that correct?